src/digitalIdentity-frontend/src/app/components/filtered-table/filtered-table.component.spec.ts

```diff
@@ -1,25 +1,200 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Comment:
Did not get coauthored eventhough I wrote a lot as well :| 
Just in a diffrent place which will not be visible as its use is deprecated.

Reply:
I'm really sorry about that :( I wanted to, but I forgot it